bot.py

```diff
@@ -474,6 +166,30 @@ async def _reply(self, ctx, content, anonymous=False):
 
         await ctx.send(embed=embed)
 
+    @commands.has_any_role(config.modRole)
```

Comment:
Providing the initial content in the open command isn't as user friendly as opening a new channel for a mod to type into. It's also entirely possible that new mods try and open a thread at the same time (due to having to type out the initial thread message).

 It would be more user friendly to open a new thread channel but not open the thread itself until the first message. Though, this does obviously have the concern of opening a channel just to have the user not have DMs open (though you could go ahead and send them a DM first-- but this might not be desired.)

Reply:
To be seen. We'll see what works better

Reply:
Conversion copied over to [Trello](https://trello.com/c/MEZ6nrCv/100#comment-5f5183efe83fb8128da7e238)